### PR TITLE
Fix #834: Landscape mode for ProfileProgressActivity

### LINKED ISSUE #834: Lowfi Landscape: ProfilProgressActivity
Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/8509e066-bfbb-4140-b329-3d43a4d20c9d/UP-User-Profile-

Also add test cases for the same.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #834 

Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/8509e066-bfbb-4140-b329-3d43a4d20c9d/UP-User-Profile-

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-land/profile_picture_activity.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:profile="http://schemas.android.com/tools">
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.profileprogress.ProfilePictureActivityViewModel" />
+    <variable
+      name="presenter"
+      type="org.oppia.app.profileprogress.ProfilePictureActivityPresenter" />
```
Comment (Luffy18346) [inline app/src/main/res/layout-land/profile_picture_activity.xml]: Remove this `presenter ` variable as it is not getting used anywhere in the file.

File: app/src/main/res/layout-land/profile_picture_activity.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+      android:id="@+id/profile_picture_image_view"
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      profile:src="@{viewModel.profileAvatar}"
+      tools:src="@drawable/ic_default_avatar"
+      app:layout_constraintBottom_toBottomOf="parent"
```
Comment (Luffy18346) [inline app/src/main/res/layout-land/profile_picture_activity.xml:22]: Formatting Issue (`profile : src` and  `tools : src` should be at bottom of the ImageView). Please press `ctrl + alt + l` to solve the issue.

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 3]
```diff
@@ -371,13 +490,75 @@ class ProfileProgressFragmentTest {
...
-    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(0)).use {
+    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(internalProfileId)).use {
+      waitForTheView(withText(R.string.stories_completed))
       onView(atPositionOnView(R.id.profile_progress_list, 0, R.id.completed_stories_container)).perform(click())
       intended(hasComponent(CompletedStoryListActivity::class.java.name))
       intended(hasExtra(CompletedStoryListActivity.COMPLETED_STORY_LIST_ACTIVITY_PROFILE_ID_KEY, internalProfileId))
     }
   }
 
+
```
Comment (Luffy18346) [inline app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt]: Remove this empty line.

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 4]
```diff
@@ -438,4 +619,46 @@ class ProfileProgressFragmentTest {
...
   }
+
+
+// TODO(#59): Move this to a general-purpose testing library that replaces all CoroutineExecutors with an
+//  Espresso-enabled executor service. This service should also allow for background threads to run in both Espresso
+//  and Robolectric to help catch potential race conditions, rather than forcing parallel execution to be sequential
+//  and immediate.
+//  NB: This also blocks on #59 to be able to actually create a test-only library.
+  /**
+   * An executor service that schedules all [Runnable]s to run asynchronously on the main thread. This is based on:
```
Comment (Luffy18346) [inline app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt:577]: Align above single line comments with this multiline comment using `tab` key

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 5]
```diff
@@ -371,13 +490,75 @@ class ProfileProgressFragmentTest {
...
+  }
+
+// TODO(#59): Remove these waits once we can ensure that the production executors are not depended on in tests.
+//  Sleeping is really bad practice in Espresso tests, and can lead to test flakiness. It shouldn't be necessary if we
+//  use a test executor service with a counting idle resource, but right now Gradle mixes dependencies such that both
+//  the test and production blocking executors are being used. The latter cannot be updated to notify Espresso of any
+//  active coroutines, so the test attempts to assert state before it's ready. This artificial delay in the Espresso
+//  thread helps to counter that.
+  /**
+   * Perform action of waiting for a specific matcher to finish. Adapted from:
```
Comment (Luffy18346) [inline app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt:474]: Align above single line comments with this multiline comment using tab key

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 6]
```diff
@@ -438,4 +619,46 @@ class ProfileProgressFragmentTest {
 
     fun inject(optionsFragmentTest: ProfileProgressFragmentTest)
   }
+
+
```
Comment (Luffy18346) [inline app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt]: remove this empty line

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 7]
```diff
@@ -184,7 +224,25 @@ class ProfileProgressFragmentTest {
   fun testProfileProgressFragmentWithProgress_recyclerViewItem0_checkOngoingTopicsCount_countIsTwo() {
     storyProgressTestHelper.markPartialTopicProgressForFractions(profileId, timestampOlderThanAWeek = false)
     storyProgressTestHelper.markTwoPartialStoryProgressForRatios(profileId, timestampOlderThanAWeek = false)
-    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(0)).use {
+    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(internalProfileId)).use {
+      waitForTheView(withText("2"))
+      onView(
+        atPositionOnView(
+          R.id.profile_progress_list,
```
Comment (Luffy18346) [inline app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt]: `atPositionOnView` method can be written in single line. Please make same correction at other places in the file.

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 8]
```diff
@@ -281,8 +362,25 @@ class ProfileProgressFragmentTest {
...
+    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(internalProfileId)).use {
       onView(withId(R.id.profile_progress_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(1))
+      waitForTheView(withText("What is a Fraction?"))
+      onView(atPositionOnView(R.id.profile_progress_list, 1, R.id.chapter_name_text_view)).check(
+        matches(
+          withText(
+            containsString("What is a Fraction?")
+          )
+        )
+      )
```
Comment (Luffy18346) [inline app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt:335]: `matches` method can be written in single line. Please make same correction at other places in file.

**Review by Luffy18346 — CHANGES_REQUESTED** [review]
LGTM thanks. Just some nit changes.

File: app/src/main/res/layout-land/profile_picture_activity.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-land/profile_picture_activity.xml]: Done.

File: app/src/main/res/layout-land/profile_picture_activity.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/layout-land/profile_picture_activity.xml:22]: Done.

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 3 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt]: Done.

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 4 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt:577]: Done

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 5 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt:474]: Done

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 6 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 7 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt]: Indentation in such cases is something which depends on developer's choice as everything is correct unless and until it crosses 80 character mark in a single line.
The main point should be that the indentation in a single file should be consistent.

File: app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 8 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt:335]: Indentation in such cases is something which depends on developer's choice as everything is correct unless and until it crosses 80 character mark in a single line.
The main point should be that the indentation in a single file should be consistent.

**Review by Luffy18346 — APPROVED** [review]
LGTM. Thanks

**Pull request merged**